tests/integration/standalone/kitty.nix

```diff
@@ -0,0 +1,73 @@
+{ pkgs, ... }: {
```

Comment:
Thanks for adding this in @rycee :pray: 

Out of curiosity (and, tbh still somewhat of a nix noob), how can I run this test on my machine (without running the entire test suite)?

Reply:
If you are using Nix Flakes then you can run it using

    nix build --option sandbox false -L .#integration-test-kitty